cli/tox.ini

```diff
@@ -23,11 +23,11 @@ commands =
 
 [testenv:py27-integration]
 commands =
-  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv -x {env:CI_FLAGS:} tests/integrations{posargs}
 
 [testenv:py34-integration]
 commands =
-  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv -x {env:CI_FLAGS:} tests/integrations{posargs}
```

Comment:
oh sorry.  I thought this was my PR.  It's helpful with local testing, but I'd rather see all test results from the async test run.

Up to you guys now though.